lstchain/scripts/onsite/onsite_create_calibration_file.py

```diff
@@ -135,16 +149,22 @@ def main():
         if ff_calibration == 'yes':
             # run lstchain script
             cmd = f"lstchain_create_calibration_file " \
-                  f"--input_file={input_file} --output_file={output_file} --pedestal_file={pedestal_file} " \
-                  f"--FlasherFlatFieldCalculator.time_calibration_path={time_file} --FlasherFlatFieldCalculator.sample_size={stat_events} "\
-                  f"--PedestalIntegrator.sample_size={stat_events}  " \
-                  f"--EventSource.max_events={max_events} --config={config_file}  >  {log_file} 2>&1"
+                  f"--input_file={input_file} --output_file={output_file} "\
+                  f"--EventSource.max_events={max_events} " \
+                  f"--LSTEventSource.EventTimeCalculator.ucts_t0_dragon={int(run_info['ucts_t0_dragon'])} " \
+                  f"--LSTEventSource.EventTimeCalculator.dragon_counter0={int(run_info['dragon_counter0'])} " \
+                  f"--LSTEventSource.EventTimeCalculator.ucts_t0_tib={int(run_info['ucts_t0_tib'])} " \
+                  f"--LSTEventSource.EventTimeCalculator.tib_counter0={int(run_info['tib_counter0'])} " \
+                  f"--LSTEventSource.LSTR0Corrections.drs4_time_calibration_path={time_file} " \
+                  f"--LSTEventSource.LSTR0Corrections.drs4_pedestal_path={pedestal_file} " \
+                  f"--FlatFieldCalculator.sample_size={stat_events} --PedestalCalculator.sample_size={stat_events} " \
+                  f"--config={config_file}  >  {log_file} 2>&1"
 
             print("\n--> RUNNING...")
             os.system(cmd)
 
             # plot and save some results
-            plot_file=f"{output_dir}/log/calibration.Run{run}.0000.pedestal.Run{ped_run}.0000.pdf"
+            plot_file=f"{output_dir}/log/calibration.Run{run}.{sub_run}.pedestal.Run{ped_run}.0000.pdf"
             print(f"\n--> PRODUCING PLOTS in {plot_file} ...")
```

Comment:
logging

Reply:
Hi @morcuended, here the idea was just to make the changes to use the new reader  (even if I actually added also some new features by mistake, so I personally was not really coherent on that).